### PR TITLE
Origin/ido

### PR DESCRIPTION
**Pull request by MichalBielicki**

### CODE REVIEW

File: src/containers/WrappedIDO/WrappedIDO.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { IDO } from '@components/IDO/IDO'
```
Comment (awojciak) [inline src/containers/WrappedIDO/WrappedIDO.tsx:1]: adding container and page wasn't in scope of task, just IDO component and storybook for it

File: src/components/IDO/style.ts [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    borderRadius: '10px',
+    marginLeft: '16px',
+    background: colors.invariant.componentOut4,
+    [theme.breakpoints.down('xs')]: {
+      minWidth: '336px',
+      marginTop: '24px',
+      marginLeft: '0px',
+      alignSelf: 'center'
+    },
+    '@media (min-width: 601px) and (max-width: 700px)': {
```
Comment (awojciak) [inline src/components/IDO/style.ts:140]: we don't have a breakpoint wit max-width equal to 700px, why is this used?

File: src/components/IDO/IDO.tsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  const [depositedXETH, setDepositedXETH] = useState<string>(' ')
+  const [depositedXBTC, setDepositedXBTC] = useState<string>(' ')
+  const [salePeriod, setSalePeriod] = useState<string>(' ')
+  const [gracePeriod, setGracePeriod] = useState<string>(' ')
+  const [SOLContributed, setSOLContributed] = useState<string>(' ')
+  const [estTokenPrice, setEstTokenPrice] = useState<string>(' ')
+  const [INVForSale, setINVForSale] = useState<string>(' ')
+
+  useEffect(() => {
+    setEst('56.0278')
```
Comment (awojciak) [inline src/components/IDO/IDO.tsx:26]: mock data should be set through props in story, not be hardcoded inside component

Comment (MichalBielicki) [reply src/components/IDO/IDO.tsx:26]: mock data i handler functions zakodowane w komponencie są tylko na potrzeby zadania, żeby odwzorować makietę

File: src/components/IDO/style.ts (hunk 2 above)
Comment (MichalBielicki) [reply src/components/IDO/style.ts:140]: [theme.breakpoints.up('xs')]: {
      width: '216px'
    }

**Review by awojciak — COMMENTED** [review]
also:
1) pull requests should go to dev branch of repo, not master
2) I can't change value in input when testing component
3)
obraz
this happens

**Pull request closed without merging**